README.md

```diff
@@ -0,0 +1,34 @@
+This repository is the source files for [http://extensions.atoum.org](http://extensions.atoum.org).
+
+The purpose is to list extension of atoum and expose it.
+
+## Adding a new extension
+
+Edit the file : `config/extensions.yml`.
+
+## Build & tests
+
+### Build
+
+```
+composer install
+```
+
+If you made some change in javascript or css, just use the [robo](http://robo.li/) build file:
+```
+vendor/bin/robo build
+```
+
+### Test
+
+```
+vendor/bin/robo test
```

Comment:
where is the robo test command ?

Reply:
oups I forget to remove it, I added it, then see that robot need to be updated then see that your libs doesn't support to be updated so roll it back ;)